concepts/05 UI Components/Form/01 Getting Started with Form/60 Modify the Form at Runtime.md

```diff
@@ -0,0 +1,175 @@
+---
+##### jQuery
+
+You can change any form, editor, or item properties at runtime. For this, use the [option(optionName, optionValue)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#optionoptionName_optionValue) method to update a Form property, and the [itemOption(id, option, value)](/Documentation/ApiReference/UI_Components/dxForm/Methods/#itemOptionid_option_value) to update the value of an item property. In the code below, we modify the [readOnly](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#readOnly) property's value:
+
+    <!-- tab: index.js -->
+    $(function() {
+        const form = $("#form").dxForm({
+            formData: {
+                // ...
+            },
+            items: [
+                // ...
+            ]
+        }).dxForm("instance");
+
+        $("#checkBox").dxCheckBox({
+            text: "Enable read-only mode",
+            value: false,
+            onValueChanged: function (e) {
+                form.option("readOnly", e.value);
+            }
+        });
+    });
+
+##### Angular
+
+You can change any form, editor, or item properties at runtime. To update a property value, bind it to a component property. In the code below, we modify the [readOnly](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#readOnly) property's value:
```

Comment:
I would apply similar edits to other fragments. I addressed the use of "we" in Teams not so long ago. 

```suggestion
You can change any form, editor, or item properties at runtime. To update a property value, bind it to a component property. The code below modifies the [readOnly](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#readOnly) property's value:
```